src/ResourceManager/Network/Commands.Network/help/New-AzureRmVirtualNetworkGateway.md

```diff
@@ -200,7 +201,33 @@ Accept wildcard characters: False
 ```yaml
 Type: Int32
 Parameter Sets: (All)
-Aliases:
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -RadiusServerAddress
+Radius server address.```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -RadiusServerSecret
+Radius server secret.```yaml
```

Comment:
Fixed. This was autogenerated  -shouldn't the generation tool take care of this?